### PR TITLE
ffprobe check to verify mp4 successfully transferred to S3 from zoom

### PR DESCRIPTION
**Pull request by nataliemona**

### CODE REVIEW

File: functions/zoom-downloader.py [hunk 1]
```diff
@@ -278,6 +281,11 @@ def stream_file_to_s3(file, uuid, track_sequence):
         s3.abort_multipart_upload(Bucket=ZOOM_VIDEOS_BUCKET, Key=filename,
                                   UploadId=mpu['UploadId'])
 
+    if file['file_type'].lower() == "mp4":
+        if not is_valid_mp4(filename):
+            stream.close()
+            raise Exception("MP4 failed to transfer.")
```
Comment (lbjay) [inline functions/zoom-downloader.py:287]: does this exception result in a DLQ entry?

File: functions/zoom-downloader.py [hunk 2]
```diff
@@ -385,3 +393,18 @@ def send_to_sqs(message, queue_name, error=None):
...
                   "FailedReason": error})
+
+
+def is_valid_mp4(filename):
+    url = s3.generate_presigned_url(
+        'get_object',
+        Params={'Bucket': ZOOM_VIDEOS_BUCKET, 'Key': filename}
+    )
+
+    command = ['/var/task/ffprobe', '-of', 'json', url]
```
Comment (lbjay) [inline functions/zoom-downloader.py:404]: did you figure out how to quiet the output? or does it matter?

File: template.yml [hunk 3]
```diff
@@ -514,10 +514,28 @@ Resources:
       Principal: events.amazonaws.com
       SourceArn: !GetAtt [ZoomNewLogGroupEventRule, Arn]
 
+  ZoomWebhookLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub "/aws/lambda/${AWS::StackName}-zoom-webhook-function"
+      RetentionInDays: 14
```
Comment (lbjay) [inline template.yml]: can you make these 60 days?

**Comment by lbjay** [comment]
I fixed your branch by just resetting it to master and cherry-picking the two commits. Not sure what happened.

File: functions/zoom-downloader.py (hunk 1 above)
Comment (nataliemona) [reply functions/zoom-downloader.py:287]: No. It raises an error and keeps the message on the queue to be retried.

File: functions/zoom-downloader.py (hunk 2 above)
Comment (nataliemona) [reply functions/zoom-downloader.py:404]: I did not figure that out. Does it matter? Would it be useful to have that output in the logs?

File: template.yml (hunk 3 above)
Comment (nataliemona) [reply template.yml]: Yes.

File: functions/zoom-downloader.py (hunk 2 above)
Comment (lbjay) [reply functions/zoom-downloader.py:404]: IDK if it's useful. I was going to try to duplicate in ipython and play with the ffprobe options and command. Just wondering if you'd done it.

**Review by lbjay — APPROVED** [review]

**Pull request merged**